### DISCUSSION TITLE
[Feature] Swap archive and like buttons on mobile client

### DISCUSSION THREAD

**Discussion by Snyaify** [immich-app/immich#2921] [Feature Request]
### The feature

Switched from photoprism to immich with my family, and allmost all of my family members when using immich mobile client the first thing to accidentally click on the "Archive photo" button.
From the point of view of ease of use, when holding a phone in your hand, it would be more logical to have a Like button in place of the Archive button, because it is more convenient to get a like button with a finger.
Honestly, I'm tired of helping family members - taking back photos from the archive :(
Please think about it. In everyday use, archiving photos is not as often necessary as putting a like

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by alextran1502** [comment]
Make sense, we can make the change

**Reply by nrgbistro** [reply]
Don't mean to rush, but is there any progress on implementing this feature request? I just experienced the same thing and discovered archiving myself by accident.

**Comment by mio-19** [comment]
> Switched from photoprism to immich with my family, and allmost all of my family members when using immich mobile client the first thing to accidentally click on the "Archive photo" button.

It was also the first thing I did!
